lib/active_admin_import/model.rb

```diff
@@ -132,6 +131,9 @@ def define_methods_for(attr_name)
       end
     end
 
+    def encode(data)
+      data.encode(force_encoding, 'binary', invalid: :replace, undef: :replace,  replace: '').sub("\xEF\xBB\xBF", "")
```

Comment:
Line is too long. [117/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.